apps/builddao/widget/components/modals/propose/ProposalVisibilityInfoModal.jsx

```diff
@@ -1,7 +1,91 @@
-const { Modal, Button } = VM.require("${config_account}/widget/components") || {
-  Modal: () => <></>,
+const { Button } = VM.require("${config_account}/widget/components") || {
   Button: () => <></>,
 };
+const { VisibilityModal } = VM.require(
+  "${config_account}/widget/components.modals.VisibilityModal",
+) || {
+  VisibilityModal: () => <></>,
+};
+
+const ModalContainer = styled.div`
+  width: 480px;
+  height: 314px;
```

Comment:
why can't we move the copytoClipboard code in this widget itself?

Reply:
Sure moved it to the widget